src/OptionSettlement.sol

```diff
@@ -56,6 +64,15 @@ contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
         claimInfo = _claim[tokenId];
     }
 
+    function claimBucket(uint256 optionId, uint16 daysAfterOptionTypeCreation)
```

Comment:
Why relative rather than absolute time here?

Reply:
switched to absolute time (in days)